source/changelog.txt

```diff
@@ -12,6 +12,25 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.0
+------
+
+*Released August 16, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1550` - Support for Range GA
+
+Bug fixes in this release:
+
+- :issue:`MONGOSH-1136` - mongosh should use stderr for password prompt
+- :issue:`MONGOSH-1425` - Expand fallback condition for $collStats to command form to account for ADF
+- :issue:`MONGOSH-1820` - 'fields' option is not working as expected in findAndModify
+- :issue:`MONGOSH-1858` - Investigate proxy e2e test failure on s390x
```

Comment:
The engineering team / @addaleax might be able to answer this, but this seems like an internal ticket and we can likely remove from public RN's.

```suggestion

```

Reply:
Removed for now. Thanks!